src/SprykerEco/Shared/Payone/Transfer/payone.transfer.xml

```diff
@@ -554,32 +584,91 @@
     </transfer>
 
     <transfer name="CheckoutResponse">
+        <property name="saveOrder" type="SaveOrder"/>
+        <property name="isExternalRedirect" type="bool"/>
+        <property name="redirectUrl" type="string"/>
+        <property name="errors" type="CheckoutError[]" singular="error"/>
+        <property name="isSuccess" type="bool"/>
     </transfer>
 
     <transfer name="SaveOrder">
+        <property name="idSalesOrder" type="int"/>
+        <property name="orderItems" type="Item[]" singular="orderItem"/>
     </transfer>
 
     <transfer name="PaymentMethods">
+        <property name="methods" type="PaymentMethod[]" singular="method"/>
     </transfer>
 
     <transfer name="CheckoutError">
+        <property name="message" type="string"/>
+        <property name="errorCode" type="int"/>
     </transfer>
 
     <transfer name="PaymentMethod">
+        <property name="methodName" type="string"/>
     </transfer>
 
     <transfer name="Customer">
+        <property name="idCustomer" type="int"/>
+        <property name="email" type="string"/>
+        <property name="dateOfBirth" type="string"/>
+        <property name="firstName" type="string"/>
+        <property name="isGuest" type="bool"/>
+        <property name="lastName" type="string"/>
+        <property name="company" type="string"/>
     </transfer>
 
     <transfer name="Address">
+        <property name="firstName" type="string"/>
+        <property name="lastName" type="string"/>
+        <property name="address1" type="string"/>
+        <property name="address2" type="string"/>
+        <property name="address3" type="string"/>
+        <property name="company" type="string"/>
+        <property name="city" type="string"/>
+        <property name="zipCode" type="string"/>
+        <property name="state" type="string"/>
+        <property name="iso2Code" type="string"/>
+        <property name="email" type="string"/>
+        <property name="phone" type="string"/>
+        <property name="salutation" type="string"/>
     </transfer>
 
     <transfer name="ShipmentMethod">
+        <property name="carrierName" type="string"/>
+        <property name="storeCurrencyPrice" type="int"/>
+        <property name="idShipmentMethod" type="int"/>
     </transfer>
 
     <transfer name="Shipment">
+        <property name="method" type="ShipmentMethod"/>
+        <property name="shipmentSelection" type="string"/>
     </transfer>
 
     <transfer name="PersonalData">
     </transfer>
+
+    <transfer name="ShipmentMethods">
+        <property name="methods" type="ShipmentMethod[]" singular="method"/>
+    </transfer>
+
+    <transfer name="Store">
+        <property name="countries" type="array" singular="country"/>
```

Comment:
Shouldn't it be like this?
`<property name="countries" type="Country[]" singular="countries"/>`

Reply:
In payone it is a simple array afaik. Legacy like

Reply:
Also singular would then be "country"
But again, this is old assoc array.